app/assets/javascripts/initializers/initializeSponsorshipVisibility.js

```diff
@@ -1,6 +1,4 @@
-'use strict';
-
-const sponsorClickHandler = event => {
+function sponsorClickHandler(event) {
```

Comment:
@maestromac, consider adding a comment above this function for now until we figure out the real solution so that someone doesn't refactor this back to `const`.